src/components/ParticipantItem/ParticipantItem.js

```diff
@@ -32,18 +32,15 @@ class ParticipantItem extends Component {
       return;
     }
     //Ensuring there is no duplicating participant
-    for (var i = 0, length = this.props.participants.length; i < length; i++) {
-      if (
-        this.state.participantName ===
-        this.props.participants[i].participantName
-      ) {
+    this.props.participants.forEach(function(participant) {
```

Comment:
let's use the ES6 arrow function instead of traditional one. For example:
```js
this.props.participants.forEach(participant => {
    //blah blah
})
```